src/frontend/src/icp/components/transactions/IcNoIndexPlaceholder.svelte

```diff
@@ -1,21 +1,28 @@
 <script lang="ts">
+	import IconAlertTriangle from '$lib/components/icons/lucide/IconAlertTriangle.svelte';
 	import IconSearchClose from '$lib/components/icons/lucide/IconSearchClose.svelte';
 	import RoundedIcon from '$lib/components/ui/RoundedIcon.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+
+	export let issueType: 'missing' | 'not-working' = 'missing';
 </script>
 
 <div class="py-12" data-tid="ic-no-index-placeholder">
 	<div class="mb-5 flex justify-center">
-		<RoundedIcon icon={IconSearchClose} />
+		<RoundedIcon icon={issueType === 'not-working' ? IconAlertTriangle : IconSearchClose} />
 	</div>
 
 	<div class="space-y-4">
 		<p class="m-0 text-center text-lg font-bold">
 			{$i18n.transactions.text.transaction_history_unavailable}
 		</p>
 		<p class="text-secondary m-0 text-center opacity-50">
-			{replaceOisyPlaceholders($i18n.transactions.text.missing_index_canister_explanation)}
+			{replaceOisyPlaceholders(
+				issueType === 'not-working'
```

Comment:
Should we add a "unit" test of its something we can easily test with E2E?

Reply:
I added a some unit tests, since the E2E tests are a bit more complex (for another PR).

I put the E2E test for this in the list of E2E tests that should be implemented